src/Tool/SqlQueryParser.php

```diff
@@ -0,0 +1,105 @@
+<?php declare(strict_types=1);
+
+namespace Manticoresearch\Buddy\Core\Tool;
+
+use PHPSQLParser\PHPSQLCreator;
+use PHPSQLParser\PHPSQLParser;
+
+class SqlQueryParser
```

Comment:
This should be the final version since we're not planning on extending this class further and we want to prevent any extensions.

Reply:
Done